PhysicsTools/NanoAOD/python/jets_cff.py

```diff
@@ -350,17 +350,22 @@
         btagHbb = Var("bDiscriminator('pfBoostedDoubleSecondaryVertexAK8BJetTags')",float,doc="Higgs to BB tagger discriminator",precision=10),
         deepTag_TvsQCD = Var("bDiscriminator('pfDeepBoostedDiscriminatorsJetTags:TvsQCD')",float,doc="DeepBoostedJet tagger top vs QCD discriminator",precision=10),
         deepTag_WvsQCD = Var("bDiscriminator('pfDeepBoostedDiscriminatorsJetTags:WvsQCD')",float,doc="DeepBoostedJet tagger W vs QCD discriminator",precision=10),
+        deepTag_ZvsQCD = Var("bDiscriminator('pfDeepBoostedDiscriminatorsJetTags:ZvsQCD')",float,doc="DeepBoostedJet tagger Z vs QCD discriminator",precision=10),
         deepTagMD_TvsQCD = Var("bDiscriminator('pfMassDecorrelatedDeepBoostedDiscriminatorsJetTags:TvsQCD')",float,doc="Mass-decorrelated DeepBoostedJet tagger top vs QCD discriminator",precision=10),
         deepTagMD_WvsQCD = Var("bDiscriminator('pfMassDecorrelatedDeepBoostedDiscriminatorsJetTags:WvsQCD')",float,doc="Mass-decorrelated DeepBoostedJet tagger W vs QCD discriminator",precision=10),
+        deepTagMD_ZvsQCD = Var("(bDiscriminator('pfMassDecorrelatedDeepBoostedJetTags:probZbb')+bDiscriminator('pfMassDecorrelatedDeepBoostedJetTags:probZcc')+bDiscriminator('pfMassDecorrelatedDeepBoostedJetTags:probZqq'))/(bDiscriminator('pfMassDecorrelatedDeepBoostedJetTags:probZbb')+bDiscriminator('pfMassDecorrelatedDeepBoostedJetTags:probZcc')+bDiscriminator('pfMassDecorrelatedDeepBoostedJetTags:probZqq')+bDiscriminator('pfMassDecorrelatedDeepBoostedJetTags:probQCDbb')+bDiscriminator('pfMassDecorrelatedDeepBoostedJetTags:probQCDcc')+bDiscriminator('pfMassDecorrelatedDeepBoostedJetTags:probQCDb')+bDiscriminator('pfMassDecorrelatedDeepBoostedJetTags:probQCDc')+bDiscriminator('pfMassDecorrelatedDeepBoostedJetTags:probQCDothers'))",float,doc="Mass-decorrelated DeepBoostedJet tagger Z vs QCD discriminator",precision=10),
         deepTagMD_ZHbbvsQCD = Var("bDiscriminator('pfMassDecorrelatedDeepBoostedDiscriminatorsJetTags:ZHbbvsQCD')",float,doc="Mass-decorrelated DeepBoostedJet tagger Z/H->bb vs QCD discriminator",precision=10),
+        deepTagMD_ZbbvsQCD = Var("(bDiscriminator('pfMassDecorrelatedDeepBoostedDiscriminatorsJetTags:probZbb'))/(bDiscriminator('pfMassDecorrelatedDeepBoostedJetTags:probZbb')+bDiscriminator('pfMassDecorrelatedDeepBoostedJetTags:probQCDbb')+bDiscriminator('pfMassDecorrelatedDeepBoostedJetTags:probQCDcc')+bDiscriminator('pfMassDecorrelatedDeepBoostedJetTags:probQCDb')+bDiscriminator('pfMassDecorrelatedDeepBoostedJetTags:probQCDc')+bDiscriminator('pfMassDecorrelatedDeepBoostedJetTags:probQCDothers'))",float,doc="Mass-decorrelated DeepBoostedJet tagger Z->bb vs QCD discriminator",precision=10),
```

Comment:
@clelange Thank you very much for the updates!

Here (and also the ones below) `pfMassDecorrelatedDeepBoostedDiscriminatorsJetTags` should be `pfMassDecorrelatedDeepBoostedJetTags` to access the `probX`

Reply:
OK, I hope it's fixed now, thanks for the careful check.